strongarm/common.py

```diff
@@ -179,3 +211,34 @@ class ListableResource(object):
     def all(cls):
         endpoint = strongarm.host + cls.endpoint
         return PaginatedResourceList(cls, endpoint)
+
+
+class CreatableResource(object):
+    """
+    A mixin for a STRONGARM resource that can be created.
+
+    The `create` method returns an instance of the newly created resource.
+
+    """
+
+    @classmethod
+    def create(cls, **kwargs):
+        endpoint = strongarm.host + cls.endpoint
+        return cls(request('post', endpoint, data=json.dumps(kwargs),
+                           headers={'Content-Type': 'application/json'}))
+
+
+class DeletableResource(object):
+    """
+    A mixin for a STRONGARM resource that can be deleted.
+
+    The `delete` method returns None on successful deletion.
+
+    """
+
+    # The attribute to be used as the unique identifier.
+    id_attr = 'id'
+
+    def delete(self, **kwargs):
+        endpoint = strongarm.host + self.endpoint + str(getattr(self, self.id_attr))
+        request('delete', endpoint)
```

Comment:
Great, so the caller at least gets some indication of success via the HTTP response code. Makes sense to me.